bluemira/equilibria/coils/_grouping.py

```diff
@@ -283,6 +284,26 @@ def from_group_vecs(cls, eqdsk: EQDSKInterface):
         pfcoils = []
         cscoils = []
         passivecoils = []
+        if eqdsk.ncoil < 1:
```

Comment:
Does a coilset not work with one coil?

Reply:
it does, it just doesnt work with 0 coils atm